### ISSUE TITLE
Baloo VFs have this quirk that affects hinting

### ISSUE THREAD

**Issue by mahalisyarifuddin** [EkType/Baloo2#7]
Greetings.

Just noticed a problem in your rather great typeface and it's around this weird looking stems. Checked all the fonts: statics, VFs, other scripts, both from this very repo and GF, and turns out it only affects all VFs, and Latin is the most problematic one, also some Indic scripts. It appears that the VFs are using "separate pieces" as issue #6 put it. It's surely affects hinting in an ugly way, and that's not a good thing.

I'm talking about this (I'm using Windows 7 and Edge 107)

![latn 400](https://user-images.githubusercontent.com/42148566/199055284-8e9f93b0-41f8-4fab-bf5d-e824331c8e01.png)
![latn 800](https://user-images.githubusercontent.com/42148566/199055313-9532e09d-f622-4c62-906a-f1c957b8aa10.png)

That's it.

**Comment by imheinrichlunge** [comment]
Yes, It's because of the way it was designed, check [this](https://i.imgur.com/N6RHQrw.png) out. This is what it looks like when you use a stroke instead of a fill.
